22.md

```diff
@@ -109,3 +110,67 @@ the client supports the [NIP-65](65.md) relay usage spec,
 ```
 
 Note that ideally there are no `p` tags unless the pubkey is being mentioned on the `.content`.
+
+### Examples for Commenting on NIP-73 External Content
+
+A podcast comment example:
+
+```js
+{
+  id: "80c48d992a38f9c445b943a9c9f1010b396676013443765750431a9004bdac05",
+  pubkey: "f7234bd4c1394dda46d09f35bd384dd30cc552ad5541990f98844fb06676e9ca",
+  kind: 1111,
+  content: "This was a great episode!",
+  tags: [
+    [
+      "o",
+      "i:podcast:item:guid:d98d189b-dc7b-45b1-8720-d4b98690f31f",
+      // note the "r" followed by a space then the url hint
+      "r https://fountain.fm/episode/z1y9TMQRuqXl2awyrQxg"
+    ],
+    // same value as "o" tag above, because it is a top-level comment (not a reply to a comment)
+    [
+      "r",
+      "i:podcast:item:guid:d98d189b-dc7b-45b1-8720-d4b98690f31f",
+      "r https://fountain.fm/episode/z1y9TMQRuqXl2awyrQxg"
+    ],
+
+    ["K", "i:podcast:item"], // uppercase K, this is the OP (original post)'s kind, that is, a podcast item
+    ["k", "i:podcast:item"] // lowercase k; in the case of a top-level comment, it is the same as the uppercase K
```

Comment:
@arthurfranca I think it's probably better to include the `guid` in this external id kind tag - so `["K", "i:podcast:item:guid"]`

Reply:
Ok, added :guid suffix.